vendor/github.com/containers/image/docs/atomic-signature-embedded-json.json

```diff
@@ -1,66 +0,0 @@
-{
```

Comment:
Please use `make vendor`, which preserves this documentation, instead of running `vndr` manually.